objectserver/engine/pack/device_io.go

```diff
@@ -282,6 +283,22 @@ func (d *PackDevice) commitLO(obj *PackObject, ot PartType) error {
 		return err
 	}
 
+	mux := common.NewKmutex()
```

Comment:
NewKmutex will create a sync.Map every time and thus it will not work as expected. Instead of creating a Kmutex abstraction, I think you can just use a global shared sync.Map  to achieve the coordination.